components/suggestion/command_provider.jsx

```diff
@@ -23,7 +29,7 @@ class CommandSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 <div className='command__title'>
-                    <string>{item.suggestion} {item.hint}</string>
+                    {item.suggestion + item.hint}
```

Comment:
Is it intentional that we're removing the space here? Also, wow `<string></string>` looks weird, although I'm not surprised it's fine from the browser's perspective

Reply:
fyi, I submitted a recent fix to remove <string> after I got JS error/warning from the browser - https://github.com/mattermost/mattermost-webapp/pull/1941

Reply:
Works fine without the space.